libfwupdplugin/fu-ble-device.c

```diff
@@ -22,50 +22,186 @@
  */
 
 typedef struct {
-	gchar		*name;
-	gchar		*address;
-	gchar		*adapter;
-	GHashTable	*characteristics;	/* utf8 : utf8 */
+	gchar			*name;
```

Comment:
I still don't see a need to keep a name attribute, get it from the parent `FuDevice`

Reply:
Fixed, thanks.